src/palm-react-native-ui-kit/screens/auth/MainAccountScreen.tsx

```diff
@@ -24,71 +26,171 @@ const MainAccountScreen = (): ReactElement => {
+                {t('Auth.CreateNewWallet')}
+              </FormText>
+            </TouchableOpacity>
+            <TouchableOpacity
+              style={styles.bottomItem}
+              onPress={(): void => {
+                navigation.navigate(Routes.RecoverAccount, {
+                  type: 'importWallet',
+                })
+              }}
+            >
+              <FormImage source={images.import} size={32} />
+              <FormText style={{ textAlign: 'center' }}>
+                {t('Auth.ImportWallet')}
+              </FormText>
+            </TouchableOpacity>
+          </Row>
+        </FormBottomSheet>
+      </>
+    )
+  }
+
+  // return (
+  //   <>
+  //     <Container style={styles.container}>
+  //       <View style={styles.top}>
+  //         <FormImage source={images.palm_logo} size={88} />
+  //         <FormText font={'B'} size={24} style={{ textAlign: 'center' }}>
+  //           {t('Auth.MainTitle')}
+  //         </FormText>
+  //       </View>
+  //       <View style={styles.bottom}>
+  //         <FormButton
+  //           onPress={(): void => {
+  //             navigation.navigate(Routes.RecoverAccount, {
+  //               type: 'restoreWallet',
+  //             })
+  //           }}
+  //         >
+  //           {t('Auth.RestoreMyAccount')}
+  //         </FormButton>
+  //         <FormButton
+  //           figure="outline"
+  //           onPress={(): void => {
+  //             setShowBottomSheet(true)
+  //             bottomSheetRef?.current?.snapToIndex(1)
+  //           }}
+  //         >
+  //           {t('Auth.SignUpWithWallet')}
+  //         </FormButton>
+  //       </View>
+  //     </Container>
+  //     <FormBottomSheet
+  //       bottomSheetRef={bottomSheetRef}
+  //       showBottomSheet={showBottomSheet}
+  //       snapPoints={snapPoints}
+  //       onClose={(): void => setShowBottomSheet(false)}
+  //     >
+  //       <Row style={styles.bottomSheet}>
+  //         <TouchableOpacity
+  //           style={styles.bottomItem}
+  //           onPress={(): void => {
+  //             navigation.navigate(Routes.NewAccount)
+  //           }}
+  //         >
+  //           <FormImage source={images.plus} size={32} />
+  //           <FormText style={{ textAlign: 'center' }}>
+  //             {t('Auth.CreateNewWallet')}
+  //           </FormText>
+  //         </TouchableOpacity>
+  //         <TouchableOpacity
+  //           style={styles.bottomItem}
+  //           onPress={(): void => {
+  //             navigation.navigate(Routes.RecoverAccount, {
+  //               type: 'importWallet',
+  //             })
+  //           }}
+  //         >
+  //           <FormImage source={images.import} size={32} />
+  //           <FormText style={{ textAlign: 'center' }}>
+  //             {t('Auth.ImportWallet')}
+  //           </FormText>
+  //         </TouchableOpacity>
+  //       </Row>
+  //     </FormBottomSheet>
+  //   </>
+  // )
```

Comment:
what is this?

Reply:
I did this to revert the code after the alphatest was over.
But it won't matter delete it.